velox/expression/ConstantExpr.cpp

```diff
@@ -162,6 +162,7 @@ void appendSqlLiteral(
     case TypeKind::TINYINT:
     case TypeKind::SMALLINT:
     case TypeKind::BIGINT:
+    case TypeKind::HUGEINT:
```

Comment:
This is to fix an issue discovered by decimal fuzzer test. Extracted this change into https://github.com/facebookincubator/velox/pull/9287.